module06/en.subject.tex

```diff
@@ -323,13 +323,13 @@ \section*{Examples}
 theta1 = np.array([2, 0.7]).reshape((-1, 1))
 gradient(x, y, theta1)
 # Output:
-array([[-19.0342574], [-586.66875564]])
+array([[-19.0342...], [-586.6687...]])
 
 # Example 1:
 theta2 = np.array([1, -0.4]).reshape((-1, 1))
 gradient(x, y, theta2)
 # Output:
-array([[-57.86823748], [-2230.12297889]])
```

Comment:
The result is exact, I have exactly the same with my code:
![image](https://user-images.githubusercontent.com/37487348/164525019-8829ccca-070e-482e-a351-6355c53b53c2.png)